Docker build errors - needs src

These are the errors running a docker build against v2.0.0 
```bash
--- github/speech-to-text-nodejs » docker build .
Sending build context to Docker daemon  38.4 MB
Step 1 : FROM centos:centos6
 ---> 8315978ceaaa
Step 2 : RUN curl -sL https://rpm.nodesource.com/setup | bash -
 ---> Using cache
 ---> 8b379921e6fa
Step 3 : RUN yum install -y nodejs  #
 ---> Using cache
 ---> df4d30864089
Step 4 : RUN npm install
 ---> Running in 10df1cb4962e
npm ERR! install Couldn't read dependencies
npm ERR! Linux 4.4.8-boot2docker
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v0.10.46
npm ERR! npm  v2.15.1
npm ERR! path /package.json
npm ERR! code ENOPACKAGEJSON
npm ERR! errno 34

npm ERR! package.json ENOENT, open '/package.json'
npm ERR! package.json This is most likely not a problem with npm itself.
npm ERR! package.json npm can't find a package.json file in your current directory.

npm ERR! Please include the following file with any support request:
npm ERR!     /npm-debug.log
The command '/bin/sh -c npm install' returned a non-zero code: 34
```
It looks like we need to re-add 
```bash
# Define working directory.
COPY ./ /src/
WORKDIR /src
``` 
to the Dockerfile

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,9 +1,10 @@
 
 FROM centos:centos6
-RUN curl -sL https://rpm.nodesource.com/setup | bash -
+RUN curl -sL https://rpm.nodesource.com/setup_6.x | bash -
 RUN yum install -y nodejs  #
 # Define working directory.
-
+COPY . /src
+WORKDIR /src
 RUN npm install
 
 EXPOSE 3000 8020

